app/manifest.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "MyWebSite",
+  "short_name": "MySite",
+  "icons": [
+    {
+      "src": "/web-app-manifest-192x192.png",
```

Not sure if we need this for the Discord integration. I don't see the images in the project. Plus we can update the name of the project.
```suggestion
      "src": "/web-app-manifest-192x192.png",
```